docs/src/main/tut/typeclasses/applicative.md

```diff
@@ -212,8 +212,7 @@ This function is provided by Cats via the `Traverse[List]` instance and syntax,
 tutorial.
 
 ```tut:book:silent
-import cats.instances.list._
-import cats.syntax.traverse._
+import cats.implicits
```

Comment:
Did this miss the `._`?

Reply:
oops, missed that, I will fix that in a separate PR.

Reply:
Why didn't this broke the build?

Reply:
the import isn't really needed for the code since it's already imported at previous blocks. I guess unused import isn't checked in tut.

Reply:
If it is not needed I'd remove it or require that each paragraph needs to specify its imports again with maybe tut's `:reset` modifier.

Reply:
I read the page again and I can see it become confusing. It looks like it would be cumbersome to reset every paragraph, I suggested that we remove `import cats.implicits._` from them and add it to the top of the page, with something like:
First, bring cats instances into the scope by
```scala
import cats.implicits._
```